.github/workflows/pip_installation.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
   loose_installation:
     name: Test loose pip installation on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+    # runs-on: ${{ matrix.os }} # `runs-on` conflicts with `uses`, see https://github.com/orgs/community/discussions/62320
```

Comment:
yup, the runs-on is decided in the `_run_tests.yml` part .. I guess you can delete this line